### PR TITLE
Allow Kotlin to auto install correctly

### PR DESCRIPTION
**Pull request by elken**
The path to the executable wasn't being found, so the package would be
downloaded, not found, and prompt the user to download it again endlessly.

### CODE REVIEW

**Review by yyoncho — APPROVED** [review]

**Comment by yyoncho** [comment]
@elken feel free to proceed with merging after getting approvals and waiting some reasonable time (1 day).

**Comment by elken** [comment]
> @elken feel free to proceed with merging after getting approvals and waiting some reasonable time (1 day).

I see, no worries! Will do in future

**Pull request merged**
